src/uikit.cs

```diff
@@ -23610,13 +24397,35 @@ interface UISheetPresentationControllerDetent {
 		[Field ("UISheetPresentationControllerAutomaticDimension")]
 		nfloat AutomaticDimension { get; }
 
+		[NoWatch, NoTV, iOS (16, 0), MacCatalyst (16,0)]
+		[Field ("UISheetPresentationControllerDetentInactive")]
+		nfloat ControllerDetentInactive { get; }
+
 		[Static]
 		[Export ("mediumDetent")]
 		UISheetPresentationControllerDetent CreateMediumDetent ();
 
 		[Static]
 		[Export ("largeDetent")]
 		UISheetPresentationControllerDetent CreateLargeDetent ();
+
+		[NoWatch, NoTV, iOS (16,0), MacCatalyst (16,0)]
+		[Static]
+		[Export ("customDetentWithIdentifier:resolver:")]
+		UISheetPresentationControllerDetent GetCustomDetent ([NullAllowed] string identifier, Func<IUISheetPresentationControllerDetentResolutionContext, nfloat> resolver);
+
+		[NoWatch, NoTV, iOS (16, 0), MacCatalyst (16,0)]
+		[Export ("identifier", ArgumentSemantic.Strong)]
+		string Identifier { get; }
+
+		[NoWatch, NoTV, iOS (16,0), MacCatalyst (16,0)]
+		[Static]
+		[Export ("customDetentWithIdentifier:resolver:")]
+		UISheetPresentationControllerDetent Create ([NullAllowed] string identifier, Func<IUISheetPresentationControllerDetentResolutionContext, nfloat> resolver);
```

Comment:
You already bound this method a few lines above with the name `GetCustomDetent`.

Reply:
🤦 
```suggestion
		UISheetPresentationControllerDetent Create ([NullAllowed] string identifier, Func<IUISheetPresentationControllerDetentResolutionContext, nfloat> resolver);
```